test/build-combination/Common/main.c

```diff
@@ -35,13 +35,11 @@
 #include "task.h"
 
 /* System application includes. */
+#include "FreeRTOSConfig.h"
```

Comment:
This should not be needed explicitly - including "FreeRTOS.h" is enough.

Reply:
+1
The first thing FreeRTOS.h does is :
~~~
/* Application specific configuration options. */
#include "FreeRTOSConfig.h"
~~~